src/components/TooltipIcon/TooltipIcon.tsx

```diff
@@ -41,6 +41,7 @@ export const TooltipIcon: FC<TooltipIconProps> = ({
                             <button
                                 type="button"
                                 data-test-id="tooltip-icon-trigger"
+                                aria-label="More info"
```

Comment:
I think it's better to expose the `aria-label` and not have it hardcoded as it might not always be "more info". Like that they can also pass a string with translation.